source/drivers/driver-compatibility-reference.txt

```diff
@@ -516,30 +516,35 @@ MongoDB Compatibility
      - MongoDB 3.0
      - MongoDB 3.2
      - MongoDB 3.4
+     - MongoDB 3.6
 
```

Comment:
Should we include a row for the 3.0 driver? Or can we generally assume it has the same compatibilities as 2.2.12?